pkgs/os-specific/linux/nvidia-x11/open.nix

```diff
@@ -18,6 +19,13 @@ stdenv.mkDerivation ({
     inherit hash;
   };
 
+  patches =
+    # Fixes framebuffer with linux 6.11
+    lib.optional (lib.versionOlder "550.40.71" nvidia_x11.version) (fetchpatch {
```

Comment:
This kind of patch will be really hard to maintain in the future, because we'll need to stop applying it once nvidia lands their own fix. For example I expect this will blow up when #344524 lands.
That is why we tend to apply patches to specific releases in `default.nix` and not in the generic builders.

I also wonder how difficult it would be to apply a patch to both the proprietary and open drivers, since patches only tend to touch their linux compat layer which is shared by both drivers.

Reply:
Oh, I see, that makes more sense. I'll fix that.